ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -87,6 +89,38 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
             this.isScrolled = false;
         }
     }
+    // Navigate through fields
+    protected nextField(event: Event, {elements, preventDefault = false, carousel = false}: NavigateFields) {
```

Comment:
**_#minor_**

It would be nice to move **_protected_** and **_private_** methods to the end of component as to have an order: 
public / protected / private